server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -21,10 +20,14 @@ class CPOnclusiveFeedParser(OnclusiveFeedParser):
     def parse(self, content, provider=None):
         onclusive_cv_items = _get_cv_items("onclusive_ingest_categories")
         anpa_categories = _get_cv_items("categories")
+        event_types = _get_cv_items("event_types")
+        onclusive_event_types = _get_cv_items("onclusive_event_types")
+
         items = super().parse(content, provider)
 
         for item in items:
             category = []
+            eventType = []
```

Comment:
would stick to snake case here - `event_type`

Reply:
sorry my bad just fixing.